OpenSearch at Berlin Buzzwords 2024 blog post (#3030)
opensearch-project/project-website: _posts/2024-07-01-opensearch-berlin-buzzwords-2024.md

meta_description: On June 9th through 11th 2024 the OpenSearch Project joined the community at Berlin Buzzwords to share knowledge and interact with the search and open source communities.
rediscoverKeywordSearch: anLaY47S_3I
flowFramework: nZdL6ykntKs

excerpt: On June 9th through 11th 2024 the OpenSearch Project joined the community at Berlin Buzzwords to share knowledge and interact with the search and open source communities.
featured_blog_post: false
featured_image: /assets/media/blog-images/2024-07-01-opensearch-berlin-buzzwords-2024/2024-0612-BBuzz-Booth.png
---  

[Berlin Buzzwords 2024](https://2024.berlinbuzzwords.de/) was a fantastic gathering for the search community again this year. The OpenSearch Project was the platinum partner again, as we feel this conference continues to be a vital place for us to meet the community. We were thankful that three of our project partners, [Eliatra](https://eliatra.com/), [Alyrn.ai](https://www.aryn.ai/), and [Adelean](https://www.adelean.com/) came to join us this year and speak to the community with us. 

> The event started off with ‘Barcamp’, a kind of “un-conference” with a schedule decided on the day. It is all driven by the interests and expertise of those who attend and this year was another great way to kick off Buzz. OpenSearch was able to share with the community multiple times during Barcamp. [Michael Froh](https://github.com/msfroh) spoke about performance optimization in Lucene, [Stavros Macrakis](https://opensearch.org/community/members/stavros-macrakis.html) spoke about search relevance, and [Hajer Bouafif](https://www.linkedin.com/in/hajerbouafif/) drove a conversation about how to choose the right ML search type for your business and she run a live demo to enrich the conversation. 

Reviewer: ```suggestion
The event started off with "Barcamp," a kind of "un-conference" with a schedule decided that day. OpenSearch was able to share with the community multiple times during Barcamp. [Michael Froh](https://github.com/msfroh) spoke about performance optimization in Lucene, [Stavros Macrakis](https://opensearch.org/community/members/stavros-macrakis.html) spoke about search relevance, and [Hajer Bouafif](https://www.linkedin.com/in/hajerbouafif/) led a conversation, including a live demo, about how to choose the right machine learning (ML) search type for your business. 
```

<img src="/assets/media/blog-images/2024-07-01-opensearch-berlin-buzzwords-2024/2024-0612-BBuzz-Barcamp.png"/>

> We had two talks at this years Buzzwords as well. 

Reviewer: ```suggestion
Additionally, we presented two talks at this year's conference. 
```

> [Praveen Mohan Prasad](https://opensearch.org/community/members/praveen-mohan-prasad.html) and [Hajer Bouafif](https://www.linkedin.com/in/hajerbouafif/) gave a talk on ‘[Rediscover your keyword search: Expand, Enrich and Rewrite](https://program.berlinbuzzwords.de/bbuzz24/talk/WLKNUS/)’, and explained that Dense Vector Search is not the only route to improve your search relevance. They shared how to empower your existing sparse keyword search with semantic search capabilities by leveraging text expansion, metadata enrichment, and query re-writing techniques. Watch the full session: 

Reviewer: ```suggestion
[Praveen Mohan Prasad](https://opensearch.org/community/members/praveen-mohan-prasad.html) and [Hajer Bouafif](https://www.linkedin.com/in/hajerbouafif/) presented [Rediscover your keyword search: Expand, Enrich and Rewrite](https://program.berlinbuzzwords.de/bbuzz24/talk/WLKNUS/) and explained why dense vector search is not the only way to improve your search relevance. They shared how to empower your existing sparse keyword search with semantic search capabilities by leveraging text expansion, metadata enrichment, and query rewriting techniques. Watch the full session: 
```

{% include youtube-player.html id=page.rediscoverKeywordSearch %}

> [Mingshi Liu](https://opensearch.org/community/members/mingshi-liu.html) and [Owais Kazi](https://opensearch.org/community/members/owais-kazi.html) presented a talk on ‘[Elevating AI Applications with OpenSearch's Flow Framework and RAG Tool](https://program.berlinbuzzwords.de/bbuzz24/talk/SRSS7M/)’ where they introduced OpenSearch's Flow Framework and RAG (Retrieval-Augmented Generation) Tool, which enables developers to create AI-augmented agent, queries, and ingestion flows, integrate ML models, and streamline app development through a no-code/low-code builder. Watch the full session: 

Reviewer: ```suggestion
[Mingshi Liu](https://opensearch.org/community/members/mingshi-liu.html) and [Owais Kazi](https://opensearch.org/community/members/owais-kazi.html) presented [Elevating AI Applications with OpenSearch's Flow Framework and RAG Tool](https://program.berlinbuzzwords.de/bbuzz24/talk/SRSS7M/), where they introduced OpenSearch's Flow Framework and Retrieval-Augmented Generation (RAG) Tool, which enables developers to create AI-augmented agents, queries, and ingestion flows, integrate ML models, and streamline app development through a no-code/low-code builder. Watch the full session: 
```

{% include youtube-player.html id=page.flowFramework %}

*[slides available here](https://github.com/opensearch-project/community/blob/main/presentations/2024-06-11-BerlinBuzzwords-Elevating-AI-Applications-Flow-Framework-RAGTool.pdf)*

* [Full playlist of presentations](https://www.youtube.com/playlist?list=PLq-odUc2x7i8jHpa6PHGzmxfAPEz-c-on)
* [Photo gallery](https://www.flickr.com/photos/newthinking_de/albums/72177720317853019/)

> While the booth was well attended on break, we noticed a few patterns; a lot of the individuals that came by either wanted to congratulate the presenters on how much they got out of the talks, or had questions on their existing workflows and instances, and came by to seek information from the experts in the booth. This told me something; OpenSearch is continuing to be adopted by the community as a platform of choice, and, it gave us ideas on further areas to develop content. If you would like to help contribute that content, check out ‘[help wanted](https://github.com/opensearch-project/project-website/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)’ and ‘[new blog](https://github.com/opensearch-project/project-website/issues?q=is%3Aissue+is%3Aopen+label%3A%22new+blog%22)’ on the project-website repository. Another observation how the booth was packed during breaks in sessions, the booth area as a whole really cleared out during the sessions. This confirms what an incredible show [Plain Schwarz](https://plainschwarz.com/) puts on, and the high value of each presentation given at Berlin Buzzwords is truly appreciated by the community. 

Reviewer: ```suggestion
Our booth was well attended, and we noticed a few patterns: A lot of people either wanted to congratulate the presenters on how much they got out of the talks or had questions about their existing workflows and instances and stopped by to seek information or advice. This told us that OpenSearch is continuing to be adopted by the community as a platform of choice, and it gave us ideas regarding further content development. If you would like to help contribute that content, check out [help wanted](https://github.com/opensearch-project/project-website/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) and [new blog](https://github.com/opensearch-project/project-website/issues?q=is%3Aissue+is%3Aopen+label%3A%22new+blog%22) in the project-website repository. We also observed that the booth was packed during breaks but that the booth area really cleared out during sessions. This confirms what an incredible show [Plain Schwarz](https://plainschwarz.com/) puts on and that each presentation given at Berlin Buzzwords is truly appreciated by the community. 
```

> While the booth was well attended on break, we noticed a few patterns; a lot of the individuals that came by either wanted to congratulate the presenters on how much they got out of the talks, or had questions on their existing workflows and instances, and came by to seek information from the experts in the booth. This told me something; OpenSearch is continuing to be adopted by the community as a platform of choice, and, it gave us ideas on further areas to develop content. If you would like to help contribute that content, check out ‘[help wanted](https://github.com/opensearch-project/project-website/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)’ and ‘[new blog](https://github.com/opensearch-project/project-website/issues?q=is%3Aissue+is%3Aopen+label%3A%22new+blog%22)’ on the project-website repository. Another observation how the booth was packed during breaks in sessions, the booth area as a whole really cleared out during the sessions. This confirms what an incredible show [Plain Schwarz](https://plainschwarz.com/) puts on, and the high value of each presentation given at Berlin Buzzwords is truly appreciated by the community. 

Reviewer: Do we want to put "project-website" in code font?

Author: good call. made that edit as well

<img src="/assets/media/blog-images/2024-07-01-opensearch-berlin-buzzwords-2024/2024-0612-BBuzz-Booth.png"/> 

> We’re already excited about next year’s Berlin Buzzwords. 

Reviewer: ```suggestion
We're already excited about next year's Berlin Buzzwords. 
```

> *Want to come connect with the OpenSearch community in person?* *Our third annual* [*OpenSearchCon North America*](https://opensearch.org/events/opensearchcon/2024/north-america/index.html) *is coming to San Francisco September 24–26, and we hope to see you there. We also recommend joining the conversation on the [forum](https://forum.opensearch.org/) and public [Slack instance](https://opensearch.org/slack.html). Interested in local User Groups? Find out if there is an event near you on our [Meetup page](https://www.meetup.com/pro/opensearchproject/), read our [charter & guidelines](https://github.com/opensearch-project/community/blob/main/user-groups/charter-guidelines.md) document, or come offer to give a talk in the [#user-groups](https://opensearch.slack.com/archives/C073A2DDP5J) channel.*

Reviewer: ```suggestion
*Want to connect with the OpenSearch community in person?* *Our third annual* [*OpenSearchCon North America*](https://opensearch.org/events/opensearchcon/2024/north-america/index.html) *is coming to San Francisco September 24–26, and we hope to see you there. We also recommend joining the conversation on the [forum](https://forum.opensearch.org/) and public [Slack instance](https://opensearch.org/slack.html). Interested in local user groups? Find out if there is an event near you on our [Meetup page](https://www.meetup.com/pro/opensearchproject/), read our [charter and guidelines](https://github.com/opensearch-project/community/blob/main/user-groups/charter-guidelines.md), or offer to give a talk in the [#user-groups](https://opensearch.slack.com/archives/C073A2DDP5J) channel.*
```
